### ISSUE TITLE
 fix enchanting table tile, add enchanting

### ISSUE THREAD

**Issue by thebigsmileXD** [ClearSkyTeam/PocketMine-MP#17]

**Comment by thebigsmileXD** [comment]
Due to plenty enchantments not being registered, the command /enchant fails and puts protection instead.

**Comment by thebigsmileXD** [comment]
The tile doesn't make trouble for me. I can open and close the inventory fine. (not use it though) @starterdragon

**Comment by starterdragon** [comment]
@thebigsmileXD yeah it can be opened and used, the problem that occurs and when you try to take the Item you have enchanted. It will be automatically dropped with the enchanment gone and with the Lapis that was used. but your XP will be reduced nonetheless

**Comment by thebigsmileXD** [comment]
@starterdragon Yea, thats due to Enchanting being unimplemented yet.

**Comment by caspervanneck** [comment]
@thebigsmileXD  when can it be implemented?

**Comment by BlawoHD** [comment]
Kommt das irgendwann?
